test/vms.encrypted.test.js

```diff
@@ -0,0 +1,202 @@
+    ip: '127.0.0.68',
+    keyId: '/foo@joyent.com/keys/id_rsa'
+};
+
+
+// --- Helpers
+
+function createOpts(path, params) {
+    return {
+        path: path,
+        headers: {
+            'x-request-id': uuid.v4(),
+            'x-context': JSON.stringify({
+                caller: CALLER,
+                params: params || {}
+            })
+        }
+    };
+}
+
+
+// --- Tests
+
+exports.setUp = function (callback) {
+    common.setUp(function (err, _client) {
+        assert.ifError(err);
+        assert.ok(_client, 'restify client');
+        client = _client;
+        callback();
+    });
+};
+
+
+// Other tests depend on there being both an 'admin' and 'external' network.
+// This test loads these and ensures we have both.
+exports.fetch_napi_networks = function (t) {
+    client.napi.get('/networks', function (err, req, res, networks) {
+        common.ifError(t, err);
+        t.equal(res.statusCode, 200, '200 OK');
+        t.ok(networks, 'networks is set');
+        t.ok(Array.isArray(networks), 'networks is Array');
+        t.ok(networks.length > 1, 'more than 1 network found');
+        var adminExtNetworks = common.extractAdminAndExternalNetwork(networks);
+        ADMIN_NETWORK = adminExtNetworks.admin;
+        EXTERNAL_NETWORK = adminExtNetworks.external;
+        t.ok(ADMIN_NETWORK, 'admin network is ' +
+            (ADMIN_NETWORK ? ADMIN_NETWORK.uuid : ADMIN_NETWORK));
+        t.ok(EXTERNAL_NETWORK, 'external network is ' +
+            (EXTERNAL_NETWORK ? EXTERNAL_NETWORK.uuid : EXTERNAL_NETWORK));
+        t.done();
+    });
+};
+
+exports.find_256M_package = function (t) {
+    client.papi.get('/packages?name=' + PACKAGE_NAME_256 + '&active=true',
+            function getPackages(err, req, res, packages) {
+        common.ifError(t, err, 'getting packages');
+        if (err) {
+            t.done();
+            return;
+        }
+
+        t.ok(packages.length > 0, 'found package named: ' + PACKAGE_NAME_256);
+        if (packages.length > 0) {
+            pkg256 = packages[0];
+        }
+
+        t.done();
+    });
+};
+
+
+exports.create_vm_256m = function (t) {
+    if (!pkg256) {
+        t.ok(false, 'Skipping - no package was found');
+        t.done();
+        return;
+    }
+
+    var vm = {
+        owner_uuid: CUSTOMER,
+        image_uuid: IMAGE,
+        networks: [ { uuid: ADMIN_NETWORK.uuid } ],
+        brand: 'joyent-minimal',
+        billing_id: pkg256.uuid,
+        encrypted: true,
+        tags: {
+            'triton.placement.exclude_virtual_servers': true
+        }
+    };
+
+    var opts = createOpts('/vms', vm);
+
+    client.post(opts, vm, function (err, req, res, body) {
+        common.ifError(t, err);
+        t.equal(res.statusCode, 202, '202 Accepted');
+        common.checkHeaders(t, res.headers);
+        t.ok(body, 'vm ok');
+
+        var jobLocation = '/jobs/' + body.job_uuid;
+        t.ok(body.job_uuid, 'body.job_uuid', 'jobLocation: ' + jobLocation);
+
```

Comment:
The t.ok() method only takes two params (not three like you have) - first is the thing to check, second is the description (additional args will be ignored) - and if you copied this from my code, I'm sorry :)

Reply:
I copied from somebody, yeah :D